src/pages/users/UserProfile.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Container, Box, Image, Center, Stack } from '@chakra-ui/react'
+import useClickCard from 'hooks/useClickCard'
+
+const UserProfile = () => {
+  const { userId } = useClickCard()
+  return (
+    <Container p={6}>
+      <Stack>
+        <Box>UserProfile</Box>
+        <Center>
+          <Box>
+            <Image
+              borderRadius="full"
+              boxSize="150px"
+              src="https://bit.ly/dan-abramov"
```

Comment:
Why this? A placeholder?

Reply:
Why would you commit testing code to prod?

Reply:
Not setup the hosting for the images yet.